certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -4784,83 +5013,183 @@
-          "400": {
-            "description": "Bad Request",
+          "202": {
+            "description": "Accepted",
             "schema": {
-              "$ref": "#/definitions/DtoResponseManageWebHook"
+              "$ref": "#/definitions/DtoResponseGetTrackedChanges"
             }
           },
-          "500": {
-            "description": "Server Error",
+          "400": {
+            "description": "Bad request",
             "schema": {
-              "$ref": "#/definitions/DtoResponseManageWebHook"
+              "$ref": "#/definitions/DtoResponseGetTrackedChanges"
             }
           }
         },
-        "x-ms-visibility": "important",
-        "summary": "When a user runs a Trigr",
-        "description": "Manages subscribing to an Encodian Trigr Webhook",
-        "x-ms-trigger": "single"
-      },
-      "x-ms-notification-content": {
-        "description": "General Response",
-        "schema": {
-          "type": "object",
-          "properties": {
-            "site": {
-              "type": "string",
-              "description": "The SharePoint site address",
-              "title": "Site Address",
-              "x-ms-visibility": "important"
-            },
-            "libraryId": {
-              "type": "string",
-              "description": "The source SharePoint Library/List ID",
-              "title": "Library/List ID",
-              "x-ms-visibility": "important"
-            },
-            "libraryName": {
-              "type": "string",
-              "description": "The source SharePoint Library/List Name",
-              "title": "Library/List Name",
-              "x-ms-visibility": "important"
-            },
-            "item": {
-              "type": "integer",
-              "format": "int32",
-              "description": "The selected document/item ID, or the first selected document/item ID from the items array.",
-              "title": "Item",
-              "x-ms-visibility": "important"
-            },
+        "x-ms-visibility": "internal",
+        "summary": "Get an Operation Status for GetTrackedChanges",
+        "description": "Retrieve the operation status of a GetTrackedChanges operation for the Operation ID provided."
+      }
+    },
+    "/api/v1/Mgmt/GetOperationStatusAddWatermark": {
+      "get": {
+        "tags": [
+          "Mgmt"
+        ],
+        "operationId": "GetOperationStatusAddWatermark",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "operationId",
+            "description": "Retrieve the operation status of an AddImageWatermark, AddImageWatermarkAdv, AddTextWatermark or AddTextWatermarkAdv operation for the Operation ID provided.",
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": { }
```

Comment:
@Encodian can you please explain why you have an empty schema? please define a proper schema.

Reply:
@Amjed-Ayoub - It is empty because there is no value passed in the body. This is exactly the same as the previously approved '/api/v1/Mgmt/GetOperationStatus***' actions, there is around 20 of them. I would request this is also approved as it matches previously approved convention.